am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/helpers/DatabaseHelperRepository.java

```diff
@@ -30,6 +31,10 @@
     @RegisterConstructorMapper(Role.class)
     Role getRole(String roleName);
 
+    @SqlQuery("select attribute from access_management "
+        + "where access_management.resource_id = :resourceId")
+    JsonPointer getAttributeForExplicitAccessRecord(String resourceId);
```

Comment:
This is wrong. Query can result in more than one row but return type is a single JsonPointer.

Please either make query specific so that only one result can be retrieved from DB or change method signature to return a list but returning `List<JsonPointer>` might not be useful / reusable in the future.

Reply:
👍